io.github.Icalingua.Icalingua.yml

```diff
@@ -0,0 +1,76 @@
+app-id: io.github.Icalingua.Icalingua
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+separate-locales: false
+rename-icon: icalingua
+rename-desktop-file: icalingua.desktop
+command: icalingua
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --share=network
+  - --persist=.pki
+  - --filesystem=xdg-documents:ro
+  - --filesystem=xdg-download
```

Comment:
Do you need add --filesystem=xdg-config/fontconfig:ro to share host's fontconfig (although chromium doesn't respect it fully)

Reply:
https://github.com/flatpak/flatpak/issues/3947
https://github.com/flatpak/flatpak/issues/3007